.github/workflows/pr-actions.yml

```diff
@@ -0,0 +1,160 @@
+name: PR actions
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions: read-all
+
+env:
+  COMMENT: ${{ github.event.comment.body }}
+  PR_NUM: ${{ github.event.issue.number }}
+  USER_EMAIL: 107717825+opentelemetrybot@users.noreply.github.com
+  USER_NAME: opentelemetrybot
+  MAX_PATCH_SIZE_KB: 1024
+
+jobs:
+  generate-patch:
+    name: Run fixer and generate patch (untrusted)
+    runs-on: ubuntu-latest
+    if: |
+      github.event.issue.pull_request &&
+      startsWith(github.event.comment.body, '/fix:')
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
I agree with @svrnm's https://github.com/open-telemetry/opentelemetry.io/pull/6894/files#r2091365013 that this isn't needed (e.g. PRs can upload artifacts, and PRs don't have any trusted permissions)

```suggestion
```